Flank/src/main/java/com/walmart/otto/filter/TestFilters.java

```diff
@@ -0,0 +1,118 @@
+package com.walmart.otto.filter;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.Arrays;
+import java.util.Collection;
+import java.util.Collections;
+import java.util.List;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+import java.util.stream.Collectors;
+
+public class TestFilters {
+
+  private static final Pattern FILTER_ARGUMENT = Pattern.compile("(.+) (.+)");
```

Comment:
It might be less error prone to do `(.+)\s+(.+) `

Reply:
Right, I'll change that